Batch File/Batch File.sublime-syntax

```diff
@@ -20,22 +22,37 @@ contexts:
       scope: keyword.control.repeat.dosbatch
     - match: \b(?:EQU|NEQ|LSS|LEQ|GTR|GEQ)\b
       scope: keyword.operator.dosbatch
-    - match: (?:^|\s)((?i)rem)(?:$|\s.*$)
-      scope: comment.line.rem.dosbatch
+    - match: "[|]"
+      scope: keyword.operator.pipe.dosbatch
+    - match: '&>|\d*>&\d*|\d*(>>|>|<)|\d*<&|\d*<>'
+      scope: keyword.operator.redirect.shell
+
+  comments:
+    - match: '(:[+=,;:])'
+      scope: punctuation.definition.comment.dosbatch
+      push:
+        - meta_scope: comment.line.colon.dosbatch
+        - match: (?=\n)
+          pop: true
```

Comment:
Recently we've been switching to line comments consuming the trailing linebreak. This is useful because the linebreak gets the `comment` scope, which allows certain features like completions from showing while typing a line comment out.